testapi.pm

```diff
@@ -78,12 +78,17 @@ Used for internal initialization, do not call from tests.
 =cut
 sub init {
     $serialdev = get_var('SERIALDEV', "ttyS0");
-    if (   get_var('OFW')
-        || check_var('BACKEND', 's390x')
-        || (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')))
-    {
+    if (get_var('OFW') || check_var('BACKEND', 's390x')) {
         $serialdev = "hvc0";
     }
+    elsif (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')) {
+        if (get_var('VERSION', '') =~ /12SP2/) {
```

Comment:
the VERSION is never 12SP2 on our instance. And the version is if anything exactly '12-SP2', so check_var will do